src/bindings/c/docs/How_to_Debug_C_API_Issues.md

```diff
@@ -0,0 +1,42 @@
+# How to Debug C API Issues
+
+C API provides exception handling, here is all possible return values of the functions:
+```ruby
+typedef enum {
```

Comment:
markdown allows to put a reference to real code ([example](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/ir/docs/support_new_opset.md)).
Please reuse this approach